.github/ISSUE_TEMPLATE.md

```diff
@@ -12,7 +12,7 @@
 Next, we need to know what your environment looks like.
 
 - Which version of OpenZeppelin are you using?
-- What network are you deploying to? testrpc? Ganache? Ropsten?
+- What network are you deploying to? ganache-cli? Ganache? Ropsten?
```

Comment:
Can we change this to just say `What network are you deploying to? Ganache? Rinkeby?`